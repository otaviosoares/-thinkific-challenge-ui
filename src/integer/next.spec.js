import React from 'react';
import { shallow, mount } from 'enzyme';
import Next from './next';

test('it renders', () => {
  var component = shallow(
    <Next />
  )
  expect(component).toMatchSnapshot();
})

test('it renders', () => {
  const mockClick = jest.fn();
  const component = mount(
    <Next onClick={mockClick}></Next>
  );
  component.find('button').simulate('click');
  expect(mockClick).toHaveBeenCalled();
})
