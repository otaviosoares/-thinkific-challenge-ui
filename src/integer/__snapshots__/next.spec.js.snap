// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  className="next"
>
  <h4>
    Get next integer
  </h4>
  <Component
    icon="arrow-right"
    style="primary"
  />
</div>
`;
